Lesson02/StartOfLesson/ToDo/Controllers/StatusController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using ToDoApp.Models;
+using ToDoApp.Services;
+
+namespace ToDoApp.Controllers
+{
+    public class StatusController : Controller
+    {
+        public ActionResult Index()
+        {
+            return View(Repository.StatusList);
+        }
+        public ActionResult Create()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Create(Status collection)
+        {
+            try
+            {
+                return RedirectToAction(nameof(Index));
+            }
+            catch
+            {
+                return View();
+            }
+        }
+        public ActionResult Edit(Status Collection)
+        {
+            return View();
```

Comment:
Then here, you want to go _get_ the record from the repository using that ID. 
```suggestion
            Status statusTheUserRequestedToEdit = Repository.StatusList.FirstOrDefault(status => status.Id == id);
            return View(statusTheUserRequestedToEdit);
```